File: Programmers/Lv1/Lv1_가운데글자가져오기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
	string answer = "";
	if (s.length() % 2 == 0)
		answer.push_back(s[(s.length() / 2) - 1]);
	answer.push_back(s[s.length() / 2]);
	return answer;
}

File: Programmers/Lv1/Lv1_가운데글자가져오기.py
def solution(s):
    answer = ''
    if(len(s)%2==0):        # 湲몄씠媛 吏앹닔
        cut = int(len(s)/2 -1)
        answer = s[cut:cut+2]
    else:               # 湲몄씠媛 ???
        cut = int((len(s)-1) / 2)
        answer = s[cut:cut + 1]

    return answer

print(solution("abcde"))